ldes-fragmentisers/ldes-pagination/src/main/java/be/vlaanderen/informatievlaanderen/vsds/PaginationStrategy.java

```diff
@@ -0,0 +1,41 @@
+package be.vlaanderen.informatievlaanderen.vsds;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.entities.LdesFragment;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.services.FragmentationStrategy;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.services.FragmentationStrategyDecorator;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.tree.member.entities.Member;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.tree.relations.TreeRelationsRepository;
+import be.vlaanderen.informatievlaanderen.vsds.services.OpenPageProvider;
+import io.micrometer.observation.Observation;
+import io.micrometer.observation.ObservationRegistry;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+
+public class PaginationStrategy extends FragmentationStrategyDecorator {
+	private final OpenPageProvider openPageProvider;
+
+	private final ObservationRegistry observationRegistry;
+
+	public PaginationStrategy(FragmentationStrategy fragmentationStrategy,
+			OpenPageProvider openPageProvider, ObservationRegistry observationRegistry,
+			TreeRelationsRepository treeRelationsRepository) {
+		super(fragmentationStrategy, treeRelationsRepository);
+		this.openPageProvider = openPageProvider;
+		this.observationRegistry = observationRegistry;
+	}
+
+	@Override
+	public void addMemberToFragment(LdesFragment parentFragment, Member member, Observation parentObservation) {
+		Observation paginationObservation = Observation.createNotStarted("pagination",
+				observationRegistry)
+				.parentObservation(parentObservation)
+				.start();
+		ImmutablePair<LdesFragment, Boolean> ldesFragment = openPageProvider
+				.retrieveOpenFragmentOrCreateNewFragment(parentFragment);
+		if (Boolean.TRUE.equals(ldesFragment.getRight())) {
```

Comment:
I would propose for readability to do a static import on Boolean.TRUE but highly optional ;)